internal/cache/market.go

```diff
@@ -81,21 +87,26 @@ func (m *MarketDescriptionCache) MarketDescriptionByID(marketID uint, variant *s
 	}
 
 	if len(missingLocales) != 0 {
-		err := m.loadAndCacheItem(missingLocales)
+		err := m.loadAndCacheItem(&marketID, variant, missingLocales)
 		if err != nil {
 			return nil, err
 		}
 
-		item, _ = m.internalCache.Get(key)
+		item, found := m.internalCache.Get(key)
+		if !found {
+			return nil, errors.Errorf("item missing key=%q", key)
+		}
+
 		result, ok = item.(*LocalizedMarketDescription)
 		if !ok {
-			return nil, errors.New("item missing")
+			return nil, errors.Errorf("unknown item type: %T with key=%q", key, item)
```

Comment:
**[POSSIBLE-BUG]:** I am not sure, but it seems, that `key` and `item` arguments are in swapped order. Please, check it.